docs/Examples/bpmn-examples/bpmn-properties.md

```diff
@@ -10,3 +10,7 @@ grand_parent: "Examples"
 ---
```

Comment:
Let's discuss how to deal with examples.

We will have multiple entry points to our examples:
https://bpmn.io/toolkit/bpmn-js/examples/
https://github.com/bpmn-io/bpmn-js-examples
And then we would another entry point: the docs.

Reply:
So I suggest keeping the examples here as we want them findable/searcheable. A user who searches for the term "theme" could find the theming example in the docs webpage. This is not the case for the examples Github repo.

One additional change I'll do is: As Philipp stated, it makes sense to include the README of the example content to the example docs pages since it'll help it to be more visible in the search results.

Reply:
I think it's okay to have all of these. GitHub is where the actual code of the examples resides. On our bpmn.io website, we use previews of these examples to give the user an idea of what might be possible with the toolkit. These previews merely link to GitHub, they could instead link to the docs. And then the docs contain the documentation part of the examples (in most cases what is now the README on GitHub) and link to the actual code on GitHub.